src/app/extractor.js

```diff
@@ -123,7 +123,7 @@ export const extractData = async (zip) => {
     console.log(`[debug] ${extractedData.channels.length} channels loaded.`);
 
     const words = extractedData.channels.map((channel) => channel.messages).flat().map((message) => message.words).flat().filter((w) => w.length > 5);
-    extractedData.favoriteWord = mostOccurences(words);
+    extractedData.favoriteWord = mostOccurences(words, true).filter((word) => !['!', '?', '/', '<', '@'].includes(word[0])).slice(0, 10).join(', ');
```

Comment:
I think that part of the fun is that the words starting with these characters are *not* excluded and excluding words starting with these characters does hurt the quality of the returned statistics.

Reply:
Yes, if we keep the check line 125 (to keep only words longer than 5) I think that we can remove these filters. The point is just to remove the non-words such as `I`, `?`, or `.`.

Reply:
That is a better solution.